### PR TITLE
Sync hasura metadata for tracking in our repo

### PR DESCRIPTION
**Pull request by LukasDeco**
We have made many changes in the hasura admin repo, ideally we can keep this repo in sync, and soon we can use the repo to make changes so they can be much more easily tracked.

### CODE REVIEW

File: packages/hasura/config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 version: 3
-endpoint: http://localhost:8080
+endpoint: https://hasura-staging-d83b.up.railway.app
```
Comment (R-K-H) [inline packages/hasura/config.yaml:2]: Is this correct or should we now start to have staging vs production branches?

Comment (LukasDeco) [reply packages/hasura/config.yaml:2]: yeah maybe we need to remove the explicit endpoint configuration, forgot I had added that.

Comment (LukasDeco) [reply packages/hasura/config.yaml:2]: actually I think we should keep this value as is. This is a required value, and localhost:8080 is the default. Can't really have environment specifics in this here.

Comment (R-K-H) [reply packages/hasura/config.yaml:2]: I see you using env vars in other places, should that not be what we do or am I missing something?

**Comment by LukasDeco** [comment]
@R-K-H this is ready for review now

**Review by R-K-H — APPROVED** [review]

**Pull request merged**